pkg/cmd/pipeline/logs.go

```diff
@@ -95,8 +95,10 @@ Show logs for given Pipeline and PipelineRun:
 	c.Flags().BoolVarP(&opts.Last, "last", "L", false, "show logs for last PipelineRun")
 	c.Flags().BoolVarP(&opts.AllSteps, "all", "a", false, "show all logs including init steps injected by tekton")
 	c.Flags().BoolVarP(&opts.Follow, "follow", "f", false, "stream live logs")
-	c.Flags().BoolVarP(&opts.Timestamps, "timestamps", "t", false, "show logs with timestamp")
+	c.Flags().BoolVarP(&opts.Timestamps, "timestamps", "", false, "show logs with timestamp")
```

Comment:
This is breaking change

Reply:
Should I remove the functionality of filtering out and printing only the mentioned tasks? If not then what flag should I use for it @piyush-garg ?